includes/dbFacile.php

```diff
@@ -420,7 +411,11 @@ function dbMakeQuery($sql, $parameters)
     krsort($namedParams);
 
     // split on question-mark and named placeholders
-    $result = preg_split('/(\?|:[a-zA-Z0-9_-]+)/', $sql, -1, (PREG_SPLIT_NO_EMPTY | PREG_SPLIT_DELIM_CAPTURE));
+    if (preg_match('/(\[\[:[\w]+:\]\])/', $sql)) {
```

Comment:
This is a 'hack' I couldn't get a preg_split regex to work across all cases so this retains existing compatibility but checks for people using mysql regex with [[:SOMETHING:]].

If someone can find a good regex to cover all basis then go for it.

It currently splits on ? and : as they are both placeholders for variable substitution and causes problems with [[:digit:]] as per issue #5118 (so fixes that)